client/src/plus/index.scss

```
@@ -50,12 +55,11 @@
         flex-direction: column;
         justify-content: center;
         align-items: center;
-        padding: 64px 32px;
         overflow: hidden;
-        box-shadow: 0px 0px 7px rgba(0, 0, 0, 0.25);
+        box-shadow: 0px 0px 7px rgba(0, 0, 0, 0.548);
```

No need for units on `0` values.

```suggestion
        box-shadow: 0 0 7px rgba(0, 0, 0, 0.548);
```